common/web/gesture-recognizer/src/engine/headless/gestures/matchers/gestureMatcher.ts

```diff
@@ -244,6 +248,15 @@ export class GestureMatcher<Type> implements PredecessorMatch<Type> {
       const matcher = this.pathMatchers[i];
       const contactSpec = this.model.contacts[i];
 
+      /* Future TODO:
+       * This should probably include "committing" the state token and items used by the subview,
+       * should they differ from the base source's original values.
+       *
+       * That said, this is only a 'polish' task, as we aren't actually relying on the base source
+       * once we've started identifying gestures.  It'll likely only matter if external users
+       * desire to utilize the recognizer.
+       */
```

Comment:
Note:  After #9803... this would mostly apply to _multitaps_; we may want to reverse the layer-write mechanism.

That said... I'm almost certain that we have no _need_ to do so; it should be a low-priority task as I don't think I'll need the corrected version for multitap handling.